frontend/src/components/projects/ProjectsDetailedContent.tsx

```diff
@@ -60,7 +67,10 @@ const ProjectsDetailedContent = ({project, studies} : ProjectsDetailedContentPro
 
 	const [activeKey, setActiveKey] = useHashURL('overview')
 
-	const currentStudy = activeKey.startsWith("study") ? Number(activeKey.split("-")[1]) : null
+	const studyTabKeyToID = useMemo(() => Object.fromEntries(studies.map((study) => [createStudyTabKey(study.id), study.id])), [studies])
```

Comment:
You should use the mapped keys as the `key` below where the TabPanes for the studies are created, just to be sure the same key prop is set on the TabPane (right now, we are calling `createStudyTabKey` to generate the keys)